simulator/src/main/java/com/hedera/block/simulator/mode/ConsumerModeHandler.java

```diff
@@ -34,31 +36,58 @@
  * {@link UnsupportedOperationException}.
  */
 public class ConsumerModeHandler implements SimulatorModeHandler {
+    /** Logger for this class */
+    private final System.Logger LOGGER = System.getLogger(getClass().getName());
 
+    // Configuration
     private final BlockStreamConfig blockStreamConfig;
```

Comment:
`BlockStreamConfig` is never read. Should it be removed?

Reply:
+1

Reply:
Removed.